test/avro_tests.erl

```diff
@@ -112,11 +112,11 @@ readme_load_schmea_test() ->
   SchemaFilename = filename:join(priv_dir(), "interop.avsc"),
   {ok, SchemaJSON} = file:read_file(SchemaFilename),
   OcfFilename = filename:join(priv_dir(), "interop.ocf"),
-  Encoder = avro:make_encoder(SchemaJSON, []),
-  Decoder = avro:make_decoder(SchemaJSON, []),
+  Encoder = avro:make_simple_encoder(SchemaJSON, []),
+  Decoder = avro:make_simple_decoder(SchemaJSON, []),
   Term = hd(element(3, avro_ocf:decode_file(OcfFilename))),
-  Encoded = iolist_to_binary(Encoder("org.apache.avro.Interop", Term)),
-  Term = Decoder("org.apache.avro.Interop", Encoded),
+  Encoded = iolist_to_binary(Encoder(Term)),
+  Term = Decoder(Encoded),
```

Comment:
I think it would worth to have a test for the equality of the `make_encoder`/`make_simple_encoder` results. Something like this:

```erlang
simple_encoder_decoder_load_schema_equality_test() ->
  SchemaFilename = filename:join(priv_dir(), "interop.avsc"),
  {ok, SchemaJSON} = file:read_file(SchemaFilename),
  OcfFilename = filename:join(priv_dir(), "interop.ocf"),
  Encoder = avro:make_encoder(SchemaJSON, []),
  Decoder = avro:make_decoder(SchemaJSON, []),
  SimpleEncoder = avro:make_simple_encoder(SchemaJSON, []),
  SimpleDecoder = avro:make_simple_decoder(SchemaJSON, []),
  Term = hd(element(3, avro_ocf:decode_file(OcfFilename))),
  Encoded = iolist_to_binary(Encoder("org.apache.avro.Interop", Term)),
  Encoded = iolist_to_binary(SimpleEncoder(Term)),
  Term = Decoder("org.apache.avro.Interop", Encoded),
  Term = SimpleDecoder( Encoded),
  ok.
```